assets/sass/_general.scss

```diff
@@ -74,13 +70,10 @@
 }
 
 .govuk-hub-comment-form__footer {
-  text-align: right;
   @include govuk-font($size: 19);
-}
-
-.govuk-hub-comment-form__footer::after,
-.govuk-hub-feedback-padding {
```

Comment:
Removing seems to have reduced the padding on the feedback widget. Class still called in `server/views/components/feedback-widget/template.njk`